src/server/journal/cmd_serializer.h

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <absl/types/span.h>
+
+#include <string>
+#include <string_view>
+
+#include "server/table.h"
+
+namespace dfly {
+
+class CmdSerializer {
+ public:
+  using Callback = std::function<void(std::string)>;
```

Comment:
How about instead calling this Callback rename to FlushSerialized?
Also Lets add some comment for this class, what it does